package.json

```diff
@@ -17,7 +17,8 @@
     "watch": "NODE_ENV=development webpack --watch --progress --config=node_modules/laravel-mix/setup/webpack.config.js",
     "hot": "NODE_ENV=development webpack-dev-server --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
     "production": "NODE_ENV=production webpack --progress --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "install": "NODE_ENV=production npm run production"
+    "install": "NODE_ENV=production npm run production",
```

Comment:
I think this will cause npm run production to run twice. We don't want that.

Reply:
So for this, I don't need the postinstall?